docs/releases/index.rst

```diff
@@ -1,12 +1,24 @@
 Release Notes
 =============
 
-The following are release notes used on PyPI, Sourceforge and mailing lists for
-Pootle releases.
+The following are release notes used on PyPI and mailing lists for Pootle
+releases.
+
+These are the changes that have happened in Pootle and may affect your server.
+Also be aware of the :ref:`important changes in the Translate Toolkit
+<toolkit:release_notes>` as many of these also affect Pootle.
+
+If you are upgrading Pootle, you might want to see some tips to ensure your
+:doc:`upgrade goes smoothly </server/upgrading>`.
 
 .. toctree::
    :maxdepth: 1
 
+   2.7.0 (development) <2.7.0>
+   2.6.0 (development) <2.6.0>
```

Comment:
Not sure it makes sense, its in the release notes and upgrade instructions already